docs/changes/utils/15777.api.rst

```diff
@@ -0,0 +1 @@
+Deprecate importing ``ErfaError`` and ``ErfaWarning`` from ``astropy.utils.exceptions``.
```

Comment:
Why not say in the change log entry where the error and warning should be imported from?

Reply:
done !